### PR TITLE
Fix Login https://github.com/timgrossmann/InstaPy/issues/1062

### PR DESCRIPTION
**Pull request by sionking**
Changes only in login file.
if Cookie file exist go with the cookies.
If not exist than wait for successful login and dump cookie to file for next login.

Thanks to @thomasuno

SC.

### CODE REVIEW

File: instapy/login_util.py [hunk 1]
```diff
@@ -2,55 +2,63 @@
 from .time_util import sleep
 from selenium.webdriver.common.action_chains import ActionChains
 from .util import update_activity
-
+import pickle
 
 def login_user(browser, username, password, switch_language=True):
     """Logins the user with the given username and password"""
-    browser.get('https://www.instagram.com')
+    browser.get('https://www.google.com')
```
Comment (Naramsim) [inline instapy/login_util.py]: Why google?

Comment (sionking) [reply instapy/login_util.py]: it is suspicious to log to Instagram with no login cookie and only after to set the cookie.
We need to act human

Comment (sionking) [reply instapy/login_util.py]: But I see I missed the Instagram get url ... I pused a fix

**Comment by timgrossmann** [comment]
@sionking This is great, I'll make sure it works as expected later today and we can merge it once some people said so as well.

**Comment by timgrossmann** [comment]
Sorry... accidentally clicked `Close and Comment`...

**Comment by Naramsim** [comment]
Hi guys, I think that would be another great idea to add to the Instapy constructor a cookie field so that if we manage to get by ourselves the cookie (for those headless), we can pass it to Instapy

`session = InstaPy(cookieFile='./whatever')`

Also, it would be good to know how the cookieFile has to be structured. I see that in the PR `pickle` was used, I don't really know how it serializes data. Maybe a better serializer could be used like the native `json` [module](https://docs.python.org/3/library/json.html). So that we can have the structure and fill it with our cookies data.

**Comment by timgrossmann** [comment]
@Naramsim Great idea!

**Comment by CharlesCCC** [comment]
I tested it today, working perfectly fine.

**Comment by JulienDev** [comment]
Hi,
Thanks for the PR! It doesn't work on Firefox : 
"selenium.common.exceptions.WebDriverException: Message: You may only set cookies for the current domain"
Also, I think it would be better to have one cookie file per user

**Comment by CharlesCCC** [comment]
@JulienDev that will be good idea, but How do you mange have multiple user within the same folder without create multiple InstaPy  folders ?

**Comment by JulienDev** [comment]
We could just have a different name for each file eg : myAccount-Cookies.pkl
I think we need it because if we try to login with cookies from another account we could have problems with the accounts

**Comment by ceribbo** [comment]
Cookies: great idea!
It would be perfect if after the first login instapy saves the cookie for each account and the keep reusing it for the next sessions until the expiration date of the cookie.

**Comment by sionking** [comment]
IMPORTANT: updated the chromedriver and chrome and I had issues with liking pages redirected me to error page. I needed to recapture the cookie file.

**Comment by converge** [comment]
#1074 #1125

**Pull request closed without merging**